lib/encryption/ssss.dart

```diff
@@ -750,6 +751,16 @@ class OpenSSSS {
       throw Exception('SSSS not unlocked');
     }
     await ssss.store(type, secret, keyId, privateKey, add: add);
+
+    // ideally never happens unless new key?
+    while (!ssss.client.accountData.containsKey(type)) {
+      Logs().d('Wait for type $type');
+      await ssss.client.oneShotSync(timeout: Duration(seconds: 3));
+    }
+    while (await getStored(type) != secret) {
+      Logs().d('Wait for secret of $type to match');
+      await ssss.client.oneShotSync(timeout: Duration(seconds: 3));
```

Comment:
What would this second loop be for?

Reply:
you could already have the key but with the old secret, the first loop is for when the key was never there

Reply:
So the first loop is redundant?

Reply:
Well `getStored` fails if there is no event with that key hence, wait for the key before checking the secret.